pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick a specific option to print to `stdout`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
+
+- Open an interactive prompt for the user to input a string with a specific placeholder:
+
+`gum input --placeholder "{{value}}"`
+
+- Interactvely pick an option which will exit if "No" is chosen:
```

Comment:
Doesn't the command exit in both cases, just with different exit values?
```suggestion
- Open an interactive confirmation prompt and exit with either `0` or `1`:
```
I know, it is not limited to confirmation prompts, but that obviously what the subcommand was meant for